### PR TITLE
AMB-186 add status healthcheck endpoint

### PR DESCRIPTION
**Pull request by RyanThomas1214**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/templates/tests.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ steps:
           -e REASONABLE_ADJUSTMENTS_BASE_URL=https://$(APIGEE_ENVIRONMENT).api.service.nhs.uk \
           -e REASONABLE_ADJUSTMENTS_PROXY_PATH=$(SERVICE_BASE_PATH) \
           -e REASONABLE_ADJUSTMENTS_PROXY_NAME=$(FULLY_QUALIFIED_SERVICE_NAME) \
+          -e STATUS_APIKEY="b3515bf1-d0a8-482f-8785-a5c0fdc19a52" \
```
Comment (artronics) [inline azure/templates/tests.yml:30]: Are you sure we want this be hardcoded here? Maybe we need to add this to secret manager?

File: api_tests/tests/reasonable_adjustments/test_happy_cases.py [hunk 2]
```diff
@@ -261,6 +261,23 @@ def test_remove_ra_record_post(self):
         # Then
         assert_that(expected_status_code).is_equal_to(response.status_code)
 
+    @pytest.mark.happy_path
+    @pytest.mark.integration
```
Comment (artronics) [inline api_tests/tests/reasonable_adjustments/test_happy_cases.py:265]: I think we use `smoke` as integration test marker. This is the first time I'm seeing this marker :)

File: azure/templates/tests.yml (hunk 1 above)
Comment (RyanThomas1214) [reply azure/templates/tests.yml:30]: spoke to Paul Walkington and he deemed this apikey non-sensitive

File: api_tests/tests/reasonable_adjustments/test_happy_cases.py (hunk 2 above)
Comment (RyanThomas1214) [reply api_tests/tests/reasonable_adjustments/test_happy_cases.py:265]: the markers `smoke` and `integration` were added as part of AMB-359, they should be in `pytest.ini`. The pipeline runs all tests marked as integration.

**Review by artronics — APPROVED** [review]

**Pull request merged**